<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetre_equip_course</class>
 <widget class="QDialog" name="fenetre_equip_course">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fenetre_equip_course</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(&quot;C:/projet-new/projet/background1.jpg&quot;);background-size:cover;
background-repeat:no-repeat;</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1161</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Equipement</string>
    </attribute>
    <widget class="QPushButton" name="excel">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>0</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Excel</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>331</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_num_equip">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Numéro de l'equipement:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_etat_equip">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Etat de l'equipement:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leNom">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>160</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_emp_equip">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Emplacement</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_equip">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_nom_equip">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Nom de l'equipement:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leEmp">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>220</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spNumero">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QComboBox" name="cbEtat">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Ouverte</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fermee</string>
       </property>
      </item>
     </widget>
     <widget class="QTextEdit" name="textType">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="leRecherche">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>10</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>recherche</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAjouterEquip">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSupprimerEquip">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>370</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QTableView" name="tabEquip">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>771</width>
       <height>301</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonModifererEquip">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>370</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbTri">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>10</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Tous</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Numéro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Etat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dimensions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Matiére</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Déscription</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_pic2">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>360</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PdfEquip">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>0</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Course</string>
    </attribute>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1161</width>
       <height>531</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="afficherCarteFid">
      <widget class="QTableView" name="tabCarteFid">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>741</width>
         <height>361</height>
        </rect>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_recherche_carteFid">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>40</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>recherche</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_colone_carteFid">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>40</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Tous</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NUM_CARTE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DATE_DE_CREATION</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>POINTS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TOTAL_ACHATS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>STATUT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>REMISE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MATRICULE_FISCALE</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="button_ajouter_carteFid">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_supprimer_carteFid">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>470</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>470</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Retour</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="ajoutClient_2">
      <widget class="QDialogButtonBox" name="validerAjout_cf">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>500</y>
         <width>193</width>
         <height>28</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>651</width>
         <height>431</height>
        </rect>
       </property>
       <property name="title">
        <string>Ajout de carte de fidélité :</string>
       </property>
       <widget class="QSpinBox" name="sb_numCarte">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="sb_points">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>110</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Num carte :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Points :</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="le_dateCreation">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>50</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>50</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date_de_creation :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Total d'achats :</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dsb_totalAchats">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>170</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Statut :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Remise :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>120</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Matricule fiscale :</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_Status">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>230</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Platinum</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Silver</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gold</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="cb_matriculeFiscale">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>120</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="sb_remise">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>290</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="modifierClient_2">
      <widget class="QDialogButtonBox" name="validerModif_cf">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>500</y>
         <width>193</width>
         <height>28</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>331</width>
         <height>351</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_num_course">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>Numéro de la course:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_tem_course">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>temp de la course :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leTacheCourse">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>160</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_tache_course">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Tache de la course :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_date_course">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>290</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>Date :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_status_course">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>Status de la course :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leStatusCourse">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>220</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spNumeroCourse">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>50</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QTimeEdit" name="cbTimeCourse">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>100</y>
          <width>118</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="textDateCourse">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>290</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="buttonModifererCourse">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>370</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonExcelCourse">
       <property name="geometry">
        <rect>
         <x>820</x>
         <y>10</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Excel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonSupprimerCourse">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>370</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonAjouterCourse">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>370</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QTableView" name="tabCourse">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>50</y>
         <width>771</width>
         <height>301</height>
        </rect>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="leRechercheCourse">
       <property name="geometry">
        <rect>
         <x>1030</x>
         <y>10</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>recherche</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbTriCourse">
       <property name="geometry">
        <rect>
         <x>920</x>
         <y>10</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Tous</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Numéro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Time</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tache</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Status</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Date</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_pic">
       <property name="geometry">
        <rect>
         <x>1030</x>
         <y>360</y>
         <width>111</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PdfCourse">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>7</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_traduction_arabe">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>version arabe</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Stat">
       <property name="geometry">
        <rect>
         <x>602</x>
         <y>7</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Statistique</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-35</y>
         <width>1181</width>
         <height>511</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <zorder>label</zorder>
      <zorder>validerModif_cf</zorder>
      <zorder>groupBox_7</zorder>
      <zorder>buttonModifererCourse</zorder>
      <zorder>ButtonExcelCourse</zorder>
      <zorder>buttonSupprimerCourse</zorder>
      <zorder>buttonAjouterCourse</zorder>
      <zorder>tabCourse</zorder>
      <zorder>leRechercheCourse</zorder>
      <zorder>cbTriCourse</zorder>
      <zorder>label_pic</zorder>
      <zorder>PdfCourse</zorder>
      <zorder>checkBox_traduction_arabe</zorder>
      <zorder>pushButton_Stat</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
